lib/parallel.rb

```diff
@@ -267,20 +271,18 @@ def work_in_processes(items, options, &blk)
               item, index = items.next
               break unless index
 
-              output = with_instrumentation item, index, options do
-                worker.work(items.pack(item, index))
-              end
-
-              if ExceptionWrapper === output
-                exception = output.exception
+              begin
+                results[index] = with_instrumentation item, index, options do
+                  worker.work(items.pack(item, index))
+                end
+              rescue StandardError => e
```

Comment:
this will no longer bubble up the DeadWorker error, not sure why this is not causing test errors ... maybe it can go away ...